docs/docs/system-administration/system-limits.md

```diff
@@ -38,3 +30,29 @@ Please contact us if you want to override the current system limits.
 
 You can modify the system limits on a per-Project basis. These limits are defined in the database against the Project.
 The easiest way to modify them is with the [Django admin](/deployment/configuration/django-admin.md) interface.
+
+## Traffic Limits
+
+### SDK Limits
+
+Requests made by our SDKs are _not_ rate limited, by design; we can't predict what sort of profile your traffic will
+look like.
+
+#### Free Plan Limits
+
+If you are on our Free Plan, we will stop answering requests from our API if you go over your
+[Free Plan limit](https://www.flagsmith.com/pricing). We will send you a warning email 7 days before we block your
+requests, to give you time to upgrade your account.
+
+#### Paid Plan Limits
+
+For paid plans, we'll give you a 30 day grace period and then charge you at $50 per million API calls for a calendar
```

Comment:
What does the grace period mean exactly? I've been trying to put it into words but not sure how it's supposed to behave.

Do you ever get more grace periods, or is it only one in the organisation's lifetime?